source/features/ci-link.tsx

```diff
@@ -38,6 +38,9 @@ void features.add({
 	include: [
 		pageDetect.isRepo
 	],
+	exclude: [
+		() => select.exists('[aria-label="Cannot fork because repository is empty."]')
```

Comment:
Why this here and the rel=canonical below?

Reply:
This runs on pages other then repo home. Only repo home will always has the rel.

Reply:
Empty repos don't have the canonical on any page